### PR TITLE
Add pod metrics

### PR DESCRIPTION
**Pull request by shuaibiyy**
- Added pod metrics
- Increased the sharpness of pod names.

screen shot 2016-12-21 at 17 02 22

### CODE REVIEW

File: app/src/pod.js [hunk 1]
```diff
@@ -132,6 +173,35 @@ export class Pod extends PIXI.Graphics {
...
+        podBox.drawRect(1, 9 - resources.cpu.requested * cpuHeight, 1, resources.cpu.requested * cpuHeight)
+        podBox.beginFill(getBarColor(resources.cpu.used, resources.cpu.limit), 1)
+        podBox.drawRect(2, 9 - resources.cpu.used * cpuHeight, 1, resources.cpu.used * cpuHeight)
+        podBox.endFill()
+        podBox.lineStyle(1, 0xaaaaaa, 1)
+
+        let limitExceeded = false
+        if (resources.memory.used > resources.memory.limit) {
+            resources.memory.limit = resources.memory.used
+            limitExceeded = true
```
Comment (shuaibiyy) [inline app/src/pod.js]: This should result in a blinking pod - signaling a possible shutdown of the pod. Figuring out how to do that.

Comment (shuaibiyy) [inline app/src/pod.js]: If this is done, most of the pods would blink because the API reports that most are using more than their limit.

**Comment by shuaibiyy** [comment]
Turns out I made a bad booboo. My unit conversion everything was in MB. Will fix.

**Comment by shuaibiyy** [comment]
Fixed. This PR is ready for review.

**Comment by hjacobs** [comment]
Thanks for the PR. Will check.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**